feat(curriculum): add chp3 md2 sharing personal details (#64546)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/es-a1-learn-contact-information-and-spelling/69370c527b6499fbfde5466f.md

---
id: 69370c527b6499fbfde5466f
title: Task 1
challengeType: 22
dashedName: task-1
lang: es
---

<!-- (Audio) Mateo: Hola, Camila. ¿Cuál es tu número de teléfono? -->

# --description--

> In this task, Mateo greets Camila and asks for her phone number. The sentence uses common vocabulary and the question word `¿Cuál?`. Pay close attention to accents, which are important in Spanish spelling and pronunciation.

Reviewer: ```suggestion
In this task, Mateo greets Camila and asks for her phone number. 

The sentence uses common vocabulary and the question word `¿Cuál?`. Pay close attention to accent marks, which are important in Spanish spelling and pronunciation.
```

# --instructions--

Listen to the audio and complete the sentence below.

# --fillInTheBlank--

## --sentence--

> `BLANK, Camila. ¿BLANK es tu BLANK de BLANK?`

Reviewer: I would suggest reducing the number of blanks.

## --blanks--

`Hola`

### --feedback--

This is the most common greeting in Spanish for "Hello."

---

`Cuál`

### --feedback--

This question word needs an accent on the "a".

---

`número`

### --feedback--

> Remember to include the accent on the "u".

Reviewer: ```suggestion
Remember to include the accent mark on the "u."
```

---

`teléfono`

### --feedback--

> This word must include the accent on the first "e".

Reviewer: ```suggestion
This word must include the accent mark on the first "e."
```
